src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVFastqUtils.java

```diff
@@ -112,10 +133,16 @@ public FastqRead read( final Kryo kryo, final Input input, final Class<FastqRead
     /** Convert a read's name into a FASTQ record sequence ID */
     public static String readToFastqSeqId( final GATKRead read, final boolean includeMappingLocation ) {
         final String nameSuffix = read.isPaired() ? (read.isFirstOfPair() ? "/1" : "/2") : "";
-        String mapLoc = "";
+        final String mapLoc;
         if ( includeMappingLocation ) {
-            if ( read.isUnmapped() ) mapLoc = " mapping=unmapped";
-            else mapLoc = " mapping=" + read.getContig() + ":" + read.getStart() + ";" + read.getCigar().toString();
+            if ( read.isUnmapped() ) {
+                mapLoc = " mapping=unmapped";
+            } else {
+                mapLoc = String.format(" mapping=%s:%d;%s;strand=%s", read.getContig(), read.getStart(), read.getCigar(),
```

Comment:
String.format is considerably slower than just gluing together the pieces with +.  But if you find it cleaner, I'm fine with it.  I don't think our code parses this information currently, but I'm guessing you probably will be parsing out the strand info.  If so, please add a method to parse the line to this class so that the formatting and the parsing live together.